applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -53,6 +53,7 @@ static struct audio_codec_info audio_codec_info[CONFIG_BT_BAP_BROADCAST_SNK_STRE
 static uint32_t bis_index_bitfields[CONFIG_BT_BAP_BROADCAST_SNK_STREAM_COUNT];
 
 static struct bt_le_per_adv_sync *pa_sync_stored;
+static uint8_t stream_stopped_reason;
```

Comment:
We need to be cautious here. Can we pass this info in line 201?

Reply:
We need to check if the reason is from TIMEOUT (lost signal from gateway) or LOCALHOST_TERM (sink drop the sync by user's input). So we need to pass this info in the beginning.